lib/legacy/templatizer-behavior.js

```diff
@@ -7,9 +7,7 @@ The complete set of contributors may be found at http://polymer.github.io/CONTRI
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 */
-import { TemplateInstanceBase as TemplateInstanceBase$0, templatize as templatize$0, modelForElement as modelForElement$0 } from '../utils/templatize.js';
-
-let TemplateInstanceBase = TemplateInstanceBase$0; // eslint-disable-line
+import { TemplateInstanceBase, templatize as templatize$0, modelForElement as modelForElement$0 } from '../utils/templatize.js';
```

Comment:
Add `// eslint-disable-line no-unused vars` at the end

Reply:
Done.